### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_Eclipse Marketplace_Keyboard: Keyboard focus is going to static content of the page

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#3364]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier! 

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.3.1; #Keyboard; #C+E-H; #Java tooling for IntelliJ/Eclipse;

**Environment Details:**
Eclipse IDE for Enterprise Java Developers
Version: 2019-06 (4.12.0)
Build id: 20190614-1200

**OS Details:** 
OS: Windows 10 (Version 1903, Build: 18362.175)
MAS Violated: MAS 1.3.1 

**NOTE: Same issue is observed in all the pop-ups**

**Repro Steps: ​**
1) Open Eclipse application
2) Tab till Help and select "Eclipse Marketplace" from the list
3) Search/Find for 'Azure Toolkit for Eclipse' and press enter
4) Check if the focus is going to the text content present on the page

**Actual Result: ​**
Keyboard focus is going to the text content present on the page

**Expected Result: ​**
Keyboard focus should not go the non-interactive content present on the page

**User Impact: ​**
Keyboard only users will find it difficult to access the page as they need to tab through each line of the page
 
**MAS Reference:**
[MAS 1.3.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4})

[Bug_3364_Eclipse_Marketplace_Text_Keyboard Focus.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451836/Bug_3364_Eclipse_Marketplace_Text_Keyboard.Focus.pptx)

**Comment by Flanker32** [comment]
@soumyatippapuram This is an external issue which has been tracked in [AAT](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), please help us verify it, thanks!

**Comment by soumyatippapuram** [comment]
Reported this issue to Eclipse (https://bugs.eclipse.org/bugs/show_bug.cgi?id=551112)

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749765
